_docs/operators_overload.md

```diff
@@ -0,0 +1,70 @@
+---
+ title: Operators overload
+ category: Data Types
+---
+
+Operator overloading allows you to redefine the behavior of operators for specific data types. This makes your code more intuitive and readable. In our language, the syntax for overloading an operator is straightforward.
+
+# Example of Overloading the == Operator
```

Comment:
you could add examples for other operators, and add a link to the appendix with a list of all operators

Reply:
added in the conclusion